backend/app/models/__init__.py

```
@@ -54,4 +54,12 @@
 
 from .threads import OpenAI_Thread, OpenAIThreadBase, OpenAIThreadCreate
 
+from .openai_conversation import (
+    OpenAI_Conversation,
```

Ruff (0.12.2)

`.openai_conversation.OpenAI_Conversation` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)